.github/workflows/update-pgxn-version.yml

```
@@ -16,21 +15,27 @@ jobs:
     name: Update pgxn configuration
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out the repo
+        uses: actions/checkout@v2
 
       - name: Install dependencies
-        run: sudo apt install libcurl4-openssl-dev libssl-dev
+        run: sudo apt-get update && sudo apt-get install libcurl4-openssl-dev libssl-dev python3-testresources
 
       - name: Clone Tools branch
-        run: git clone --depth 1 --branch v0.8.1 https://github.com/citusdata/tools.git
+        run: git clone --depth 1 --branch update_properties_fixes https://github.com/citusdata/tools.git
 
       - name: Install Python requirements
         run: python -m pip install -r tools/packaging_automation/requirements.txt
 
+      - name: Set git name and email
+        run: |
+          git config --global user.email "${{ github.event.inputs.microsoft_email }}" && \
+          git config --global user.name "${{ github.event.inputs.name }}"
+
       - name: Update pgxn files
         run: |
-          python -m packaging_automation.update_pgxn \
-          --prj_ver ${PROJECT_VERSION} \
+          python -m tools.packaging_automation.update_pgxn \
+          --prj_ver ${{ github.event.inputs.project_version }} \
           --gh_token ${GH_TOKEN} \
```

Why do you not have quotation marks here?

```suggestion
          --gh_token "${GH_TOKEN}" \
```

Please go through all the file for similar issues.

---

Fixed